### PR TITLE
Made civvie research less grindy.

### PR DESCRIPTION
**Pull request by Ophanimdev**
Reduced civilian research prices all around because it's rather ridiculous to have to pay 10k research points just to figure out what a MICROWAVE is. 

This should make the civilian research grindset a lot shorter and therefore empower people to actually enjoy the game rather than fondle an obese, latex-wearing, unemployed western Canadian's """vision"""

### CODE REVIEW

File: Research/industrial.yml [hunk 1]
```diff
@@ -129,7 +129,7 @@
     state: icon
   discipline: Industrial
   tier: 2
-  cost: 7500
+  cost: 500 #You can get most of these items from vending machines and there's like 50 of them in a given machine
```
Comment (MLGTASTICa) [inline Research/industrial.yml]: This is a vital research. Bring it back to 7.5k

File: Research/civilianservices.yml [hunk 2]
```diff
@@ -134,7 +134,7 @@
     state: display
   discipline: CivilianServices
   tier: 1
-  cost: 7500
+  cost: 6666 #Made for centisyzing feotuses
```
Comment (MLGTASTICa) [inline Research/civilianservices.yml]: remove this comment , against github tos bro

File: Research/industrial.yml [hunk 3]
```diff
@@ -143,7 +143,7 @@
     state: base
   discipline: Industrial
   tier: 2
-  cost: 10000
+  cost: 2500
```
Comment (MLGTASTICa) [inline Research/industrial.yml]: keep at original cost. this unlocks thrusters

**Review by MLGTASTICa — CHANGES_REQUESTED** [review]

**Comment by Ophanimdev** [comment]
Well, I was under the impression that R&D as is would be eventually scrapped and is more of a placeholder feature at the moment. But, very well.

File: _Crescent/Entities/Objects/Misc/liquidcontainers.yml [hunk 4]
```diff
@@ -89,6 +89,7 @@
   parent: FuelCan
   name: boriatic fuel canister
   id: JugBoriaticFuel
+  description: Petroleum has always been produced from organic beings. This liquid has always been made from the fear, pleasure, grief, loathing, and resentment of living creatures!
```
Comment (MLGTASTICa) [inline _Crescent/Entities/Objects/Misc/liquidcontainers.yml]: boriatic fuel is not petroluem , lore check fail!!!

**Review by MLGTASTICa — CHANGES_REQUESTED** [review]

**Comment by Ophanimdev** [comment]
It's done.
But could you clarify the nature of boriatic fuel? The word doesn't seem to actually mean anything and the only reference I could find to it were the following links:
https://tardis.fandom.com/wiki/Boriatic_Wastes
https://forums.spacebattles.com/threads/sector-crescent-hullrot.1179364/

**Comment by Boltersam** [comment]
> It's done. But could you clarify the nature of boriatic fuel? The word doesn't seem to actually mean anything and the only reference I could find to it were the following links: https://tardis.fandom.com/wiki/Boriatic_Wastes https://forums.spacebattles.com/threads/sector-crescent-hullrot.1179364/

It's a volatile liquid fuel that explodes when the container/engine is damaged, or ignites when a puddle is heated - effectively space gasoline. It's the most ubiquitous and cheap form of fuel, usually for small or jury-rigged ships

Actual nature & origin is undefined since fuel refining was removed to be readded later, name is gibberish

**Pull request merged**